### ISSUE TITLE
Question about XGTE

### ISSUE THREAD

**Issue by dungeonguru** [Sklore/HL_DD_5e_Colab#783]
So, when I create/copy the UA subclasses, I was planning on changing the name to add " - Depreciated" and then adding the tag "Helper.Obsolete" to prevent being able to pick it for a new character. I guess I should make sure that is what we want to do. I can leave the subclass live (no Helper.Obsolete), but I don't want people asking for us to support the outdated class option because they like it better than the final version.

Part 2 of the question is what do we want to do with sources that get completely emptied out? The UA - Barbarian source is completely superseded by Xanathar's. If we "Helper.Obsolete" all the subclasses, there's no point in keeping the source live - so I was thinking of Hiding them? 

Input anyone?

**Comment by kjbro** [comment]
Naming them deprecated makes sense, as does obsoleting them.  Favored Soul will be particularly interesting for this since there were already 2 or 3 versions of that one live in the pack (should probably deprecate all of them now?).  

Hiding the barbarian source but leaving it in the pack for characters with the deprecated versions also makes sense.

TL;DR my input is you've got the the right plan

**Comment by LordMorin** [comment]
Personally, I'd leave them as they are. If they are unwanted by a player or
DM, then its just a matter of unchecking the item under Configure Hero.
Broadband link - error
The Bell Internet connection is currently unavailable

Error: Your Bell Internet connection is currently unavailable
Please check the lights on your modem to ensure both the *DSL* and
*INTERNET* lights are solid green.

[image: Bell device] If both the *DSL* and *INTERNET* lights are not solid
green after approximately 10 minutes click below to reboot your modem.

After rebooting your modem, wait 10 minutes before checking the lights on
your modem.

If the *DSL* and *INTERNET* lights are still not solid green, please call
our 24/7 toll-free technical support line at 310-SURF (310-7873).




<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
